eofconvention.fsx

```diff
@@ -0,0 +1,23 @@
+open System.IO
+
+let EOFIsEOL(path: string) = 
```

Comment:
@realmarv nit, this is a bad practice called "Primitive Obsession" (look it up for more info); to fix it you should use the type FileInfo instead of string

Reply:
Done